e2e/cases/asset/svgr/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "asset-svgr-test",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "dependencies": {
+    "react": "^18.3.1"
```

Comment:
should `'react'` in dev dep?

Reply:
react should external

Reply:
peerDep + devDep?

Reply:
As we usually don't make a library deps on `'react'` directly.

Reply:
peer is enough? We do not use react directly